codeclimate-wrapper

```diff
@@ -41,17 +41,34 @@ else
     && invalid_docker_host "$DOCKER_HOST"
 fi
 
+CC_XDG_CONFIG_HOME=${XDG_CONFIG_HOME:-~/.config/}
+CC_XDG_CONFIG="${CC_XDG_CONFIG_HOME}/codeclimate/config.yml"
```

Comment:
These variables aren't themselves related to XDG: it's our config file, it could live anywhere, and we're nice enough to follow XDG in our default paths. Also, why not let the users set alternate values if they want?

```sh
: "${CC_CONFIG:=$XDG_CONFIG_HOME/codeclimate/config.yml}"
: "${CC_CACHE:=$XDG_CACHE_HOME/codeclimate/cache.yml}"
```